### PR TITLE
Documentation for the Party element.

### PR DESCRIPTION
**Pull request by jdmgoogle**
This addresses #201.

### CODE REVIEW

File: docs/xml/elements/party.rst [hunk 1]
```diff
@@ -1,35 +1,35 @@
...
++---------------------+---------------------------+-----------+----------+----------------------+-------------------------------+
+| Tag                 | Data Type                 | Required? | Repeats? |Description           |Error Handling                 |
+|                     |                           |           |          |                      |                               |
++=====================+===========================+===========+==========+======================+===============================+
+| Abbreviation        | xs:string                 | Optional  | Single   |An abbreviation for   |If the field is invalid or not |
+|                     |                           |           |          |the party name.       |present, the implementation is |
+|                     |                           |           |          |                      |required to ignore it.         |
++---------------------+---------------------------+-----------+----------+----------------------+-------------------------------+
+| Color               | HtmlColorString           | Optional  | Single   |The preferred display |If the field is invalid or not |
```
Comment (jungshadow) [inline docs/xml/elements/party.rst]: Do we define `HtmlColorString` elsewhere? I forget.

**Comment by jungshadow** [comment]
Other than the one question, LGTM. /cc @pstenbjorn @cjerdonek

File: docs/xml/elements/party.rst (hunk 1 above)
Comment (jdmgoogle) [inline docs/xml/elements/party.rst]: No. Not sure we do `TimeWithZone`, either. We should probably fix that. I'll file issues.

**Comment by jdmgoogle** [comment]
Filed #215 to track documentation for `HtmlColorString`.

**Comment by jungshadow** [comment]
@jdmgoogle Do you feel we should just treat `HtmlColorString` like `LanguageString` in that we folded the latter into `InternationalizedText`? I'm not against tracking it elsewhere, but it may be overkill to create a new section of types if their uses are isolated.

**Comment by jdmgoogle** [comment]
I'd be okay with that; my only concern is that if we end up using it elsewhere in the future we'll need to move it, but I guess that's a relatively minor issue.

I'll make the change by adding `HtmlColorString` to `Party` and `TimeWithZone` to `HoursOpen`.

**Comment by jdmgoogle** [comment]
Latest commit addresses `HtmlColorString` for #215.

**Comment by jungshadow** [comment]
LGTM yet again.

**Comment by jdmgoogle** [comment]
Thanks.

@pstenbjorn @cjerdonek  @nomadaisy @pkoms

**Comment by pstenbjorn** [comment]
LGTM

**Pull request merged**